nomad/observability.nomad

```diff
@@ -25,7 +25,8 @@ job "otel-collector" {
 
       # Receivers
       port "otlp-grpc" {
-        to = 4317
+        to     = 4317
+        static = 4317
```

Comment:
What's the significance of this change?

Reply:
Basically, the previous version meant

"There is a named port named `otlp-grpc`. In the container, that port points at :4317. From the outside, I make no promises about which port number `otlp-grpc` is served from - it's dynamically chosen." 

The new version means
"There is a named port named `otlp-grpc`. In the container, that port points at :4317. From the outside, I make a promise that you can access it at {attr.unique.network.ip-address}:4317"